### ISSUE TITLE
Regarding ease of setup/Clear Documentation

### ISSUE THREAD

**Issue by logan-keede** [openzim/wp1#712]
I am Prospective GSOC 2024 contributor, since it was required that to get kiwix project we must have at least one PR, I looked for a good first issue, and I believe I could solve the  #491 if I could actually setup the developer and testing environment , I tried reading the Readme file and following instruction, but it was not very effective, documentation is vague at many points.
I know these projects are not made with beginners in mind but I hope setup could be simplified somewhat, otherwise label "Good first issue" is a bit misleading.

PS:- it seems the documentation is made for linux, but it is not explicitly mentioned anywhere,(even though it should not have much effect except use of replacement commands)  i will try to set it up again in linux, and hopefully resolve #491

**Comment by audiodude** [comment]
Thanks for the feedback. I can definitely add a note that the instructions are for Linux, that might help. Otherwise, I'd need to know what parts of the documentation (I assume you mean `README.md`) are unclear in order to fix anything.

**Comment by audiodude** [comment]
Please let me know if the changes I've made are helpful at all.

**Comment by logan-keede** [comment]
It seems all my problem arise from the fact that I was trying to install this on Windows, I will try installation on Linux let you know if I face any problems. There is a bit of confusion in **credentials.py.dev** but i am assuming i just need to copy from **credentials.py.dev.example** and add key, secret in **STORAGE** for which i need to contact @kelson42 and get a developer account in zimfarm, (or is there an other way?), what is the relevance of **replica.my.conf** mentioned in Readme.md? is it strictly needed for installation? if so, how? where can i find it?

**Comment by logan-keede** [comment]
> Please let me know if the changes I've made are helpful at all.

Yes, changes are helpful, but consider adding what is needed to be added manually, or what is already present in the setup, for example 
> You also will need two databases in your server: `enwp10_test` and `enwikip_test`.
do I need to create this database myself or are they already present after applying migrations for dev docker, 
also you have mentioned.
>If you've followed the steps under 'Development' below to create a running dev database with docker-compose, you're all set
do I need to follow similar steps for test environment setup, or are you suggesting running pytest directly on dev instance instead of setting up different testing environment

**Comment by audiodude** [comment]
Will be fixed by 1054
